libs/dh/metering-point/feature-overview/src/components/measurement/dh-measurements.component_v2.ts

```diff
@@ -75,7 +75,17 @@ import { DhMeasurementsFilterComponent } from './dh-measurements-filter.componen
   `,
 })
 export class DhMeasurementsV2Component {
+  private transloco = inject(TranslocoService);
   private locale: WattSupportedLocales = inject(LOCALE_ID) as WattSupportedLocales;
+  private sum = computed(
+    () =>
+      `${this.formatNumber(this.measurements().reduce((acc, x) => acc + x.current.quantity, 0))} ${this.unit()}`
+  );
+  private unit = computed(() =>
+    this.transloco.translate(
+      'meteringPoint.measurements.units.' + this.measurements()[0]?.current.unit
+    )
```

Comment:
suggestion: let's return en empty string in case `this.measurements()[0]?.current.unit` is `undefined` (happens while the date is fetching). So we avoid a console error

![image](https://github.com/user-attachments/assets/0a318f15-1c3f-45dd-9497-70d6157c8f5b)

Reply:
Good idea